src/wdio/browser.ts

```diff
@@ -12,9 +12,21 @@ export class Browser {
 
 
     // Keyboard special keys
+    public static async pressEsc(): Promise<void> {
+        return browser.keys(['Escape']);
+    }
     public static async pressTab(): Promise<void> {
         return browser.keys(['Tab']);
     }
+    public static async pressBackspace(): Promise<void> {
+        return browser.keys(['Backspace']);
+    }
+    public static async pressEnter(): Promise<void> {
+        return browser.keys(['Enter']);
+    }
+    public static async pressSupr(): Promise<void> {
```

Comment:
I would name this one as "pressDelete", considering that "Supr" is the wording of Spanish keyboards, so we should go to a more generic approach